modules/tinymce/src/core/test/ts/browser/dom/SelectionQuirksTest.ts

```diff
@@ -64,8 +65,20 @@ describe('browser.tinymce.core.dom.SelectionQuirksTest', () => {
     TinySelections.setSelection(editor, [ 0, 0, 0 ], 0, [ 0, 0 ], 0);
     TinyContentActions.keyup(editor, Keys.left(), { shift: true });
     assertNormalizeCounter(0);
-    TinyContentActions.keyup(editor, 17, { }); // single ctrl
+    TinyContentActions.keyup(editor, 17, { }); // Single Ctrl
     assertNormalizeCounter(1);
     TinyAssertions.assertSelection(editor, [ 0, 0 ], 0, [ 0, 0 ], 0);
   });
+
+  it('TINY-4550: Normalization should not run after selecting all with keyboard shortcut when there is only an image in the content', () => {
```

Comment:
```suggestion
  it('TINY-4550: Normalization should not run after selecting all when there is only an image in the content', () => {
```
This should happen regardless of how the select all is triggered, so this just helps clarify that I think.

Reply:
Also should we have a test for `mouseup` as well (or similar)?